WordPress/Classes/ViewRelated/Post/PostListFilter.swift

```diff
@@ -68,7 +68,7 @@ import Foundation
         let filterType: Status = .published
         let statuses: [BasePost.Status] = [.publish, .publishPrivate]
 
-        let predicate = NSPredicate(format: "status IN %@", statuses.strings)
+        let predicate = NSPredicate(format: "postID > 0 AND status IN %@", statuses.strings)
```

Comment:
I know you explain why this is needed like 15 lines below in `draftFilter` comment, but adding a short line like "see predicate on `draftFilter()` on why this is needed would be great — when debugging things, it's easy to develop a tunnel vision and miss things like that :)

Reply:
Thanks for pointing it out. I also found a small bug here. Fixed and updated the comment in 4c13ac72896785c6d46dd8f1ed2ecf437477603b.